1_1_vulns/TrainingDataPoisoning.md

```diff
@@ -4,41 +4,48 @@
 7. Testing and Detection, by measuring the loss during the training stage and analyzing trained models to detect signs of a poisoning attack by analyzing model behavior on specific test inputs.
-   1. Monitoring and alerting on the number of skewed responses exceeding a threshold.
+   1. Monitoring and alerting on number of skewed responses exceeding a threshold.
    2. Use of a human loop to review responses and auditing.
-   3. Implement dedicated LLMs to benchmark against undesired consequences and train other LLMs using [reinforcement learning techniques](https://wandb.ai/ayush-thakur/Intro-RLAIF/reports/An-Introduction-to-Training-LLMs-Using-Reinforcement-Learning-From-Human-Feedback-RLHF---VmlldzozMzYyNjcy).
+   3. Implement dedicated LLM's to benchmark against undesired consequences and train other LLM's using [reinforcement learning techniques](https://wandb.ai/ayush-thakur/Intro-RLAIF/reports/An-Introduction-to-Training-LLMs-Using-Reinforcement-Learning-From-Human-Feedback-RLHF---VmlldzozMzYyNjcy).
    4. Perform LLM-based [red team exercises](https://www.anthropic.com/index/red-teaming-language-models-to-reduce-harms-methods-scaling-behaviors-and-lessons-learned) or [LLM vulnerability scanning](https://github.com/leondz/garak) into the testing phases of the LLM's lifecycle.
 
 **Example Attack Scenarios:**
 
 1. The LLM generative AI prompt output can mislead users of the application which can lead to biased opinions, followings or even worse, hate crimes etc.
 2. If the training data is not correctly filtered and|or sanitized, a malicious user of the application may try to influence and inject toxic data into the model for it to adapt to the biased and false data.
-3. A malicious actor or competitor intentionally creates inaccurate or malicious documents that are targeted at a model’s training data which is training the model at the same time based on inputs. The victim model trains using this falsified information which is reflected in the outputs of generative AI prompts to its consumers.
-4. The vulnerability [Prompt Injection](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/blob/main/1_0_vulns/PromptInjection.md) could be an attack vector to this vulnerability if insufficient sanitization and filtering are performed when client input to the application are used to train the model. I.E, if malicious or falsified data is input to the model from a client as part of a prompt injection technique, this could inherently be portrayed in the model data.
+3. A malicious actor or competitor intentionally creates inaccurate or malicious documents which are targeted at a model’s training data in which is training the model at the same time based on inputs. The victim model trains using this falsified information which is reflected in outputs of generative AI prompts to it's consumers.
```

Comment:
Possible rewording for readability?
A malicious actor or competitor intentionally creates inaccurate or malicious documents, specifically targeting the training data used to train a model. The victim model then incorporates this misleading information, affecting the outputs it produces in response to generative AI prompts for its users.